docs/snippets/composition-environments/xrd.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xvpcs.aws.platformref.upbound.io
+spec:
+  group: aws.platformref.upbound.io
+  names:
+    kind: XVPC
+    plural: xvpcs
+  versions:
+  - name: v1alpha1
+    served: true
+    referenceable: true
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              vpcName:
+                type: string
+            required:
+            - vpcName
```

Comment:
Please also add file with XR https://raw.githubusercontent.com/ytsarev/composition-environment-example/main/xvpc.yaml for easy end-to-end testing

Reply:
Added a `claim.yaml`